zerver/views/streams.py

```diff
@@ -206,13 +207,13 @@ def compose_views(request, user_profile, method_kwarg_pairs):
     return json_success(json_dict)
 
 @has_request_variables
-def remove_subscriptions_backend(request, user_profile,
-                                 streams_raw = REQ("subscriptions", validator=check_list(check_string)),
-                                 principals = REQ(validator=check_list(check_string), default=None)):
-    # type: (HttpRequest, UserProfile, Iterable[Text], Optional[Iterable[Text]]) -> HttpResponse
+def remove_subscriptions_backend(
+        request: HttpRequest, user_profile: UserProfile,
+        streams_raw: Iterable[Text]=REQ("subscriptions", validator=check_list(check_string)),
+        principals: Optional[Iterable[Text]]=REQ(validator=check_list(check_string),default=None)
+) -> HttpResponse:
 
-    removing_someone_else = principals and \
```

Comment:
Was not sure about this line break being there, so ended up rectifying it to one line, if this is an issue I'll put it back